pyhive/hive.py

```diff
@@ -69,7 +71,8 @@ class Connection(object):
     """Wraps a Thrift session"""
 
     def __init__(self, host, port=10000, username=None, database='default', auth='NONE',
-                 configuration=None, kerberos_service_name=None, password=None):
+                 configuration=None, kerberos_service_name=None, password=None,
+                 mode="NONE", http_path=None):
```

Comment:
- Can you change `mode` to `transport_mode` to better match the Hive setting and be more obvious?
- Please add some docs:
  - `transport_mode` corresponds to hive.server2.transport.mode
  - `http_path` corresponds to hive.server2.thrift.http.path